### PR TITLE
Fix typos in localization strings

### PR DESCRIPTION
**Pull request by danyaPostfactum**
Some minor typos I found in your plugin when translating. Unfortunately, I can't figure out how to rebuild `pot` file..

### CODE REVIEW

File: includes/class-woo-wallet-payment-method.php [hunk 1]
```diff
@@ -59,14 +59,14 @@ public function init_form_fields() {
             'title' => array(
                 'title' => __('Title', 'woo-wallet'),
                 'type' => 'text',
-                'description' => __('Payment method description that the customer will see on your checkout.', 'woo-wallet'),
```
Comment (danyaPostfactum) [inline includes/class-woo-wallet-payment-method.php]: This typo comes from woocommerce COD payment gateway

File: includes/class-woo-wallet.php [hunk 2]
```diff
@@ -266,7 +266,7 @@ public function locate_template($template_name, $template_path = '', $default_pa
      */
     public function admin_notices() {
         echo '<div class="error"><p>';
-        _e('WooCommerce Wallet plugin requires <a href="http://wordpress.org/extend/plugins/woocommerce/">WooCommerce</a> plugins to be active!', 'woo-wallet');
```
Comment (danyaPostfactum) [inline includes/class-woo-wallet.php]: This url is outdated

Comment (malsubrata) [reply includes/class-woo-wallet.php]: Oops

**Comment by malsubrata** [comment]
Thanks for your continued support.

**Comment by malsubrata** [comment]
Regenerate POT file is so easy with current plugin setup with grunt cli. 
If you have not install grunt cli then please flow this [link](https://gruntjs.com/getting-started) to install. After that type `grunt makepot` in the terminal to regenerate POT file.
Please feel free to ask me anything regarding this.

**Comment by danyaPostfactum** [comment]
`grunt makepot` works nice, thanks, I will use that command for future fixes.

**Pull request merged**